pkg/network/openshift_sdn.go

```diff
@@ -141,14 +142,6 @@ func fillOpenShiftSDNDefaults(conf, previous *operv1.NetworkSpec, hostMTU int) {
 		conf.KubeProxyConfig.ProxyArguments = map[string][]string{}
 	}
 
-	if conf.KubeProxyConfig.ProxyArguments["metrics-bind-address"] == nil {
-		conf.KubeProxyConfig.ProxyArguments["metrics-bind-address"] = []string{"0.0.0.0"}
```

Comment:
Double-checking: because this used to be in the defaults, we previously would have serialized a last-applied-configuration that contained these values explicitly. When the user upgrades to 4.2, we will no longer be adding those values to the conf, and so CNO will perceive their configuration as having changed. However, this won't matter because we allow all kube-proxy config changes, so we're good.

Right?

Reply:
That was my thinking. I actually have a branch with some tests to make sure we don't violate this for a theoretical existing cluster.